.github/workflows/main.yml

```
@@ -10,67 +10,105 @@ on:
 
 jobs:
   test:
-    name: ${{ matrix.task }} (${{ matrix.os }}, ${{ matrix.compiler }}, R ${{ matrix.r_version }})
+    name: ${{ matrix.task }} (${{ matrix.os }}, ${{ matrix.compiler }}, R ${{ matrix.r_version }}, ${{ matrix.build_type }})
     runs-on: ${{ matrix.os }}
     strategy:
       fail-fast: false
       matrix:
         include:
           - os: ubuntu-latest
+          ################
+          # CMake builds #
+          ################
```

Oops, I was wrong with this suggestion in my previous review.
```suggestion
          ################
          # CMake builds #
          ################
          - os: ubuntu-latest
```

---

ha! I missed it, thanks.